openedx/features/caliper_tracking/tests/expected/edx.course.student_notes.added.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Tagged",
+  "actor": {
+    "id": "http://localhost:18000/u/edx",
+    "name": "edx",
+    "type": "Person"
+  },
+  "eventTime": "2018-12-13T09:55:11.309Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36",
+      "event_source": "browser",
+      "event_type": "edx.course.student_notes.added",
+      "host": "8c3b43aa7b1b",
+      "ip": "172.18.0.1",
+      "org_id": "edx",
+      "page": "http://localhost:18000/courses/course-v1:edx+CS0110+2014_T/courseware/f1fd55ec037e468ba39c24e998ab1327/82f15a955a8d4af8aedb8ed599da324f/1?activate_block_id=block-v1%3Aedx%2BCS0110%2B2014_T%2Btype%40vertical%2Bblock%40f6de5da157854652b39c9ac5a1ddfd2d",
+      "path": "/event",
+      "session": "95e42e97e3b23f545334167a1acc5256",
+      "user_id": 2
+    }
+  },
+  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
+  "object": {
+    "extensions": {
+      "component_usage_id": "block-v1:edx+CS0110+2014_T+type@html+block@9fd0bcf6454b436bb221a97b796381ef",
+      "course_id": "course-v1:edx+CS0110+2014_T",
+      "highlighted_content": "shown",
+      "note_id": "5",
+      "note_text": "Made it. ",
+      "tags": [],
+      "truncated": []
+    },
+    "id": "http://localhost:18000/courses/course-v1:edx+CS0110+2014_T/courseware/f1fd55ec037e468ba39c24e998ab1327/82f15a955a8d4af8aedb8ed599da324f/1?activate_block_id=block-v1%3Aedx%2BCS0110%2B2014_T%2Btype%40vertical%2Bblock%40f6de5da157854652b39c9ac5a1ddfd2d",
+    "type": "Annotation"
```

Comment:
The Object `type` should be `DigitalResource` as per caliper documentation.

Reply:
done